### DISCUSSION TITLE
Setting up Callsign Lookup

### DISCUSSION THREAD

**Discussion by ve3hls** [magicbug/Cloudlog#1025] [Q&A] [answered]
I see that Cloudlog can look up station on QRZ and extract some basic information without a subscription.  U.S. State is all I'm after at the moment.  

It looks like I have to add or edit some lines in a file called config.php in order to do this.  I have no idea where this file is.

Can you help?

73,

Ken, VE3HLS

**Comment by magicbug** [comment, marked as answer]
Already setup for lookups Ken hence why when you type in new callsigns it populates all the fields if its available.

It doesn't do it on import at the moment but you can check your log using https://ve3hls.cloudlog.co.uk/index.php/update/check_missing_grid 

It will be slow and probably time out so you'll need to run a it a few times and it will fill in missing grids but not states, I can add that though.

**Reply by ofadam** [reply]
States/counties lookup would be amazing!

**Comment by ve3hls** [comment]
Hi Peter,

I get a blank page with "0 updated" in the upper left corner when I click
that link.  Don't know what it means.

I'll just keep using QRZ.

Thanks,

Ken  VE3HLS

**Comment by magicbug** [comment]
Thats good means everything has Gridsquare, I'll add US States to the same script over the weekend, although I've noticed its not always the best but gives people the option.

**Comment by ve3hls** [comment]
That's great! Thanks very much Peter.

The way I use Cloudlog; all of my logs are entered as .adif files, whether
from WSJT or the N1MM contest logger.

Some day I'll be back on satellites when I can afford a 9700!

73,

Ken  VE3HLS
So Phisai, Thailand
Blog:  bueng-ken.com
